docs/source/traverse_config_key.rst

```diff
@@ -0,0 +1,4 @@
+ .. note::
```

Comment:
Docs for `config` are now (even before this PR) inconsistent:

![image](https://user-images.githubusercontent.com/3383210/198826415-2d7ea1c3-faf0-470d-81c5-e8a3a1888bbd.png)

Some parts as Note, some as plain text. I'd unify towards **plain text**. Can be done on a separate PR ofc

Reply:
Hmm, yes, it's not perfect, the Note format is being used since we wanted to store the doc text in one place and use it across several commands as a Note, will have to see if there is a straight forward way to convert Note to plain text -> https://github.com/VirtusLab/git-machete/issues/719